<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmXvFlowConfig</class>
 <widget class="QWidget" name="FrmXvFlowConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="vSpacerTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="XMatLabel" name="lbFlowLoopInterval">
       <property name="text">
        <string>循环间隔(ms):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="XMatSpinBox" name="spbFlowLoopInterval">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="XMatCheckBox" name="ckbFuncErrorInterruptRun">
       <property name="text">
        <string>算子错误中断运行</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XMatCheckBox</class>
   <extends>QAbstractButton</extends>
   <header>XMatCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>XMatSpinBox</class>
   <extends>QSpinBox</extends>
   <header>XMatSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>XMatLabel</class>
   <extends>QLabel</extends>
   <header>XMatLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
